codersdk/agentsdk/convert_test.go

```diff
@@ -62,6 +63,7 @@ func TestManifest(t *testing.T) {
 					Threshold: 22555666,
 				},
 				Health: codersdk.WorkspaceAppHealthInitializing,
+				Hidden: true,
```

Comment:
I spotted [this](https://github.com/coder/terraform-provider-coder/pull/276#discussion_r1746644281) while reviewing this test.

Probably worth checking here as well.

Reply:
It appears the test has multiple conflicting fields already so I don't think this is an issue here.

Reply:
SGTM 👍 Thanks for checking.